scripts/linters/custom_eslint_checks/rules/e2e-browser-nested-awaits.js

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint to ensure that e2e tests wrap
+   `browser.switchTo().activeElement()` in an `await` expression
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: (
+        '`browser.switchTo().activeElement()` ' +
+        'should be wrapped in an `await` expression'),
+      category: 'Possible Errors',
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      noBrowserAwait: (
+        '`browser.switchTo().activeElement()` ' +
+        'is not wrapped in an `await` statement')
+    },
+  },
+
+  create: function(context) {
+    /*  Checks if CallExpression `expr` is syntactically the same as:
+     *  `browser.switchTo().activeElement()`.
+     *
+     *  If it is, checks to see whether `expr` is wrapped
+     *  in an `await` expression.
+     */
+    var checkExprNode = function(expr) {
+      var node = expr.callee;
+      if (node.type !== 'MemberExpression' ||
+        node.property.type !== 'Identifier' ||
+        node.property.name !== 'activeElement' ||
+        node.object.type !== 'CallExpression') {
+        return false;
+      }
+
+      node = node.object.callee;
+      if (node.type !== 'MemberExpression' ||
+        node.property.type !== 'Identifier' ||
+        node.property.name !== 'switchTo' ||
+        node.object.type !== 'Identifier' ||
+        node.object.name !== 'browser') {
+        return false;
+      }
```

Comment:
Please use the [selector](https://eslint.org/docs/developer-guide/selectors#listening-for-selectors-in-rules) to make it simple. Let me know if you need any help! :)

Reply:
As far as I'm aware, there is no selector for siblings of parents, so I can't use one selector to match this pattern. I need to match an `Identifier` and `CallExpression` under a `MemberExpression` parent node, which isn't possible with ESLint's current selectors. For reference, here is the structure I am trying to match (can't match the things in red):
![image](https://user-images.githubusercontent.com/70606586/119404550-f4f51a80-bcad-11eb-87fc-e999744781ff.png)

I could also be misunderstanding your suggestion/wrong, so please let me know if there is a way to do this.